AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -824,7 +830,7 @@ - (void)removeDownloadedReleaseDetailsIfUpdated:(NSString *)currentInstalledRele
   [MS_USER_DEFAULTS removeObjectForKey:kMSDownloadedReleaseHashKey];
 }
 
-- (nullable NSMutableDictionary *)getReportingParametersForUpdatedRelease:(nullable NSString *)updateToken
+- (nullable NSMutableDictionary *)getReportingParametersForUpdatedRelease:(BOOL)isPublic
```

Comment:
Good time to reconsider the method name.
My suggestion would be `generateReportingParametersForReleaseWithCurrentInstalledReleaseHash:distributionGroupID:publicTrack:`.

Reply:
Let's avoid this refactoring.